sdk/python/feast/permissions/client/utils.py

```diff
@@ -16,6 +16,8 @@ def create_auth_header(
 
 def create_flight_call_options(auth_config: AuthConfig) -> fl.FlightCallOptions:
     if auth_config.type != AuthType.NONE.value:
-        headers = create_auth_header(auth_config)
+        auth_client_manager = get_auth_client_manager(auth_config)
```

Comment:
Why did you change this ? it looks the same to me only the encoding part, I think you should change create_auth_header with the logic you wrote here

Reply:
Like this

```
def create_auth_header(
    auth_config: AuthConfig,
) -> list[tuple[bytes, bytes]]:
    auth_client_manager = get_auth_client_manager(auth_config)
    token = auth_client_manager.get_token()

    return [(b"authorization", b"Bearer " + token.encode("utf-8"))]
```